app/konnect/dev-portal/applications/dynamic-client-registration/okta.md

```diff
@@ -57,27 +57,98 @@ In order to map an application from the Dev Portal to Okta, you have to create a
 
-2. Click **Settings** to open the Dev Portal settings.
+1. Log in to {{site.konnect_short_name}} and select the Dev Portal {% konnect_icon dev-portal %} from the menu.
 
-3. Click the **Application Setup** tab to open the DCR settings for your Dev Portal.
+2. Navigate to **Application Auth** for the authentication settings of your API Products.
 
-4. Enter the **Issuer URL** for your authorization server, and the **Token** that were created in Okta.
+3. Open the **DCR Providers** to see all existing DCR providers.
 
-5. Enter the names of the **Scopes** and **Claims** as comma-separated values in their corresponding fields. The values should match the scopes or claims that were created in Okta.
+4. Select **New DCR Provider** to create a new Okta configuration. Name it for internal reference within {{site.konnect_short_name}}. This name and the provider type won't be visible to developers on the Dev Portal.
+
+5. Enter the **Issuer URL** of your authorization server and the **DCR Token** that you created in Okta. Select Okta as the **Provider Type**. The DCR token will be stored in isolated, encrypted storage and will not be readable through any Konnect API.
+
+6. Save your DCR provider. You should now see it in the list of DCR providers.
+
+7. Navigate to the **Auth Strategy** tab, then select **New Auth Strategy** to create an auth strategy that uses the DCR provider.
+
+8. Provide a name for internal use within {{site.konnect_short_name}} and a display name for visibility on your Portal. In the **Auth Type** dropdown menu select DCR. In the **DCR Provider** dropdown, select the name of the DCR provider config you just created. Your **Issuer URL** will be prepopulated with the Issuer URL you added to the DCR provider.
+
+9. Enter the names of the **Scopes** and **Claims** as comma-separated values in their corresponding fields. The values should match the scopes or claims that were created in Okta.
 
    {:.note}
-   > **Note:** You can use any of the existing scopes besides **`openid`**, as using the `openid`
-   scope prevents you from using client credentials. If the **Scopes** field is empty, `openid`
-   will be used.
+   > **Note:**  Avoid using the `openid` scope with client credentials as it restricts the use. If no scopes are specified, `openid` will be the default.
+
+10. Select the relevant **Auth Methods** you need (`client_credentials`, bearer, session), and **Save**. 
 
-6. Click **Save**.
+{% endnavtab %}
+{% navtab API %}
+After configuring Okta, you can integrate it with the Dev Portal for dynamic client registration (DCR). This process involves two steps: creating the DCR provider and establishing the authentication strategy. DCR providers are designed to be reusable configurations. This means once you've configured the Okta DCR provider, it can be utilized across multiple authentication strategies without needing to be set up again.
```

Comment:
```suggestion
After configuring Okta, you can integrate it with the Dev Portal for dynamic client registration (DCR). This process involves two steps: creating the DCR provider and establishing the authentication strategy. DCR providers are designed to be reusable configurations. This means once you've configured the Okta DCR provider, it can be used across multiple authentication strategies without needing to be set up again.
```